Source/WebCore/page/writing-tools/WritingToolsTypes.h

```diff
@@ -47,6 +47,10 @@ enum class Action : uint8_t {
     Restart,
 };
 
+enum class RequestedTool : uint8_t {
```

Comment:
Is this big enough?

Reply:
Looks like it is for now, but no guarantee in the future.

Reply:
Can we just use a larger type?